content/modules/redisgears/jvm/install.md

```diff
@@ -0,0 +1,98 @@
+---
+Title: Install RedisGears and the JVM plugin 
+linkTitle: Install 
+description:
+weight: 20
+alwaysopen: false
+toc: "true"
+categories: ["Modules"]
+---
+
+Before you can use RedisGears with the JVM, you need to install the RedisGears module and JVM plugin on your Redis Enterprise cluster and enable them for a database.
+
+## Prerequisites
+
+1. Redis Enterprise v6.0.0 or later
+
+1. Created a Redis Enterprise cluster
+
+1. Added nodes to the cluster
+
+## Install RedisGears and the JVM plugin
+
+If your cluster uses Redis Enterprise v6.0.12 or later, you only need to download the RedisGears package. The JVM plugin will install automatically if you make a `POST` request to the `/v2/modules` REST API endpoint.
+
+For earlier versions of Redis Enterprise, you need to download both the RedisGears and the JVM plugin packages and use a `POST /v1/modules` request.
+
+After the install is complete, RedisGears will appear in the list of available modules on the Redis Enterprise admin console's **settings** and **create database** pages.
+
+### Redis Enterprise v6.0.12 or later
+
+1. Download the RedisGears package from the [download center](https://redis.com/redis-enterprise-software/download-center/modules/).
+
+1. Upload the package to a node in the cluster.
+
+1. Add RedisGears to the cluster with a `POST` request to the `/v2/modules` REST API endpoint:
+
+    ```sh
+    $ curl -k -u "<user>:<password>" -F "module=@/tmp/redisgears.linux-centos7-x64.1.2.1.zip" https://localhost:9443/v2/modules
+    ```
+
+{{<note>}}
+The `POST /v2/modules` request automatically downloads the RedisGears JVM plugin as a dependency.
+{{</note>}}
+
+### Redis Enterprise v6.0.8 or earlier
```

Comment:
I believe we can say that we are not supporting this version, will make things easier, @K-Jo WDYT?

Reply:
It will.  let's do so @MeirShpilraien @rrelledge